tests/utils.py

```diff
@@ -297,3 +304,23 @@ def get_resource(version: int, key: str = "key1", agent: str = "agent1", value:
         "purged": False,
         "requires": [],
     }
+
+
+@lru_cache(1)
+def get_product_meta_data() -> ProductMetadata:
+    """Get the produce meta-data"""
+    bootloader = InmantaBootloader()
+    context = bootloader.load_slices()
+    return context.get_product_metadata()
+
+
+def product_version_lower_than(version: str) -> bool:
+    return parse_version(get_product_meta_data().version) < parse_version(version)
+
+
+def mark_only_for_version_higher_than(version: str) -> "MarkDecorator":
```

Comment:
This name indicates that the test case will run for all versions strictly larger than the given version, but looking at the implementation, it seems that the test case will also run when the versions match. I think the implementation is incorrect. Shouldn't we skip the test case when the given version is smaller then or equal to the given version?

Reply:
oh my, good catch, thx